tests/scripts/multi-node/build-rook.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+set -e
+
+
+#############
+# VARIABLES #
+#############
+
+rook_git_root=$(git rev-parse --show-toplevel)
+rook_kube_templates_dir="$rook_git_root/cluster/examples/kubernetes/"
+
+
+#############
+# FUNCTIONS #
+#############
+
+function fail_if {
+  if [[ "$(uname -n)" != "k8s-01" ]]; then
+    echo "You must run the script from k8s-01"
+    exit 1
+  fi
+  if [[ "$(id -nu)" != "vagrant" ]]; then
+    echo "Wrong user, log as 'vagrant'"
+    exit 1
+  fi
+  if ! git rev-parse --show-toplevel &> /dev/null; then
+    echo "It looks like you are NOT in a Git repository"
+    echo "This script should be executed from WITHIN Rook's git repository"
+    exit 1
+  fi
+}
+
+function purge_rook_pods {
```

Comment:
if we only have to build on the host, this function may have value outside of this script, so that any kubectl pointing at the right cluster can be used to clean up Rook from anywhere.

Reply:
Done.